### ISSUE TITLE
add breadcrumbs

### ISSUE THREAD

**Issue by OlivierJM** [SparkEdUAB/sparked-client#41]
For a better user experience, we can have breadcrumbs for navigation on the dashboard
something like this

course > units > topics > resources

**Comment by YoramGondwe** [comment]
Should these bread crumbs be clickable so that one can go back to a particular page by clicking them?

**Comment by OlivierJM** [comment]
Yeah it should have some auto routing and keeping the history in mind.
